Character.cpp

```diff
@@ -86,8 +86,19 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
+void boost(int& current, int& initial)
+{
+    if(current < initial)
+    {
+        current = initial;
+
+        current *= 1.1;
+        initial = current;  
+    }
```

Comment:
This is not following the spec
```
        When you defeat another Character: 
            a) your stats are restored to their initial value if they are lower than it.	            
            b) your stats are boosted 10%	            
            c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.	            
```
your stats are ALWAYS boosted by 10%
the initial value of your stat is ALWAYS updated to reflect the boosted stat.

Only the first clause `a)` happens when `current < initial`.
clauses `b)` and `c)` happen **regardless** of if `current < initial`
They should not be a part of the `if( )` body.
- [ ] revise